hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/TssLibrary.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.tss;
```

Comment:
If the goal is to start building something in parallel that can later be replaced with the actual library, I would suggest respecting packages and naming in the library, so when replaced, the client code is not prone to being refactored by the change of location and naming.

Reply:
The initial goal is to have a fake implementation to be used for integration testing in `test-clients`. Just pushed [here](https://github.com/hashgraph/hedera-services/pull/15908/commits/5d9c15375e81ae5b7c85a8f5ac6454d4e0acaeb8) what I currently have in progress, please review again and let me know of any further feedback.